spring-cloud-aws-autoconfigure/src/main/java/org/springframework/cloud/aws/autoconfigure/context/ContextStackAutoConfiguration.java

```diff
@@ -33,6 +33,7 @@
  * @author Agim Emruli
  */
 @Configuration
+@ConditionalOnAwsCloudEnvironment
```

Comment:
No, I don't think that Gitter would help somehow.
It's even much better that we have a code scope in this Pr already, that may simplify a discussion.

So, this change is definitely wrong.
See source code of this  `ContextStackAutoConfiguration` class, especially part like:
```
@ConditionalOnProperty(prefix = "cloud.aws", name = "stack.auto", havingValue = "true", matchIfMissing = true)
@ConditionalOnAwsCloudEnvironment
@ConditionalOnMissingBean(StackResourceRegistryFactoryBean.class)
public static class StackAutoDetectConfiguration extends ContextStackConfiguration {
```
So, we have already `@ConditionalOnAwsCloudEnvironment` there and it is enough IMO.